transport/http/src/main/java/org/kaazing/gateway/transport/http/HttpAcceptor.java

```diff
@@ -593,6 +594,21 @@ public void removeBridgeFilters(IoFilterChain filterChain) {
         }
     }
 
+    private static HttpSubjectSecurityFilter newSubjectSecurityFilter(Logger logger) {
+        Class<HttpSubjectSecurityFilter> clazz = HttpSubjectSecurityFilter.class;
+        ServiceLoader<HttpSubjectSecurityFilter> loader = ServiceLoader.load(clazz);
```

Comment:
ServiceLoader mechanism is expensive as it scans all the jars in the classpath. So keep a field for it in this class. Then call iterator() in this method.